daemon/images/metrics.go

```diff
@@ -1,18 +0,0 @@
-package images // import "github.com/docker/docker/daemon/images"
-
-import (
-	metrics "github.com/docker/go-metrics"
-)
-
-// ImageActions measures the time it takes to process some image actions.
-// Exported for use in the containerd-backed image store and it is not intended
-// for external consumption. Do not use!
-var ImageActions metrics.LabeledTimer
-
-func init() {
-	ns := metrics.NewNamespace("engine", "daemon", nil)
-	ImageActions = ns.NewLabeledTimer("image_actions", "The number of seconds it takes to process each image action", "action")
-	// TODO: is it OK to register a namespace with the same name? Or does this
-	// need to be exported from somewhere?
-	metrics.Register(ns)
```

Comment:
Went looking if the `metrics.Register` was still needed, but look like this one is also registered in `daemon/metrics.go`;

https://github.com/moby/moby/blob/a72026acbbdfcf90f0ba203abd4e0943e3d546e7/daemon/metrics.go#L19
https://github.com/moby/moby/blob/a72026acbbdfcf90f0ba203abd4e0943e3d546e7/daemon/metrics.go#L57

Reply:
Yeah, it's not needed because the namespace is already registered